conf/nginx/heroku.conf.php

```diff
@@ -18,6 +18,9 @@
 
     fastcgi_buffers 256 4k;
 
+    # define a disk cache for this nginx instance
+    proxy_cache_path /var/tmp/nginx_cache levels=1:2 keys_zone=DYNOCACHE:16m inactive=24h max_size=512m;
```

Comment:
i would have done this:

proxy_cache_path <?=getcwd()?>/nginxcache levels=1:2 keys_zone=DYNOCACHE:60m inactive=24h max_size=512m;